src/client/app/components/search/searchResults.js

```diff
@@ -28,15 +28,15 @@ const SearchResults = ({ searchResults, playlistId, dispatch }) => {
                 className="search-results__grid-wrapper"
             >
                 {searchResults && searchResults.map((tile, i) => {
-                    const { artist, title, album, images } = tile;
+                    const { artist, title, album, images, releaseDate } = tile;
                     const image = images.length && images[1] ?
                         images[1] : { url: 'http://placehold.it/300x300' };
 
                     return (
                         <GridTile
                             key={i}
                             title={title}
-                            subtitle={<span>by <b>{artist}</b></span>}
+                            subtitle={<div><p>by <b>{artist}</b></p><p>on <b>{album}</b> released {releaseDate}</p></div>}
```

Comment:
Small thing, but can this subtitle be extracted into its own functional component, if for no other reason that readability?

Reply:
Tried it out! Pushed my first React component ever (credit to @ericconner5 for his code school experience & support) lmk if that is what you meant.

Reply:
I don't need to add any tests for it, do I? Since it doesn't do anything but exist.